buf.gen.yaml

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env -S go run github.com/bufbuild/buf/cmd/buf generate -o internal/testpb internal/testpb/test.proto --template
+---
+version: "v1"
+plugins:
+  - name: 'go'
+    path: ["go", "run", "google.golang.org/protobuf/cmd/protoc-gen-go"]
+    out: '.'
+    opt: 'paths=source_relative'
+  - name: 'go-grpc'
+    out: '.'
+    path: ["go", "run", "google.golang.org/grpc/cmd/protoc-gen-go-grpc"]
+    opt: 'paths=source_relative'
```

Comment:
Probably want to add a CI step that asserts the generated code is correct, but I don't suspect this will change often.

Reply:
yeah that's right, given it's for tests, I didn't feel like spending CI cycles on it